Notes/Codes.md

```diff
@@ -2,6 +2,8 @@
 
 ## Sparse vs Dense
 
+If a code makes use of less than or equal to half of its available codewords its said to be sparse, otherwise the code is  dense. Binary codes have $2^N$ available codewords for $N$ bits so, in this case, codes that have $ \le 2^{N - 1}$ codewords would be deemed sparse.
```

Comment:
Thanks for this! Still not sure if this is actually a real thing or only invented for the purposes of this course